.github/workflows/build.yml

```diff
@@ -316,3 +316,101 @@ jobs:
           conda activate $pwd\tmp_env
           $env:path = "$pwd\build\bin\Release;" + $env:path
           .github\scripts\windows\test_full_pipeline.ps1 -Backend trt -Device cuda
+
+  badge_build_cpu_model_convert:
```

Comment:
@huayuan4396  could we put this step and the end of job `build_cpu_model_convert` as last step?